authors/apps/articles/urls.py

```diff
@@ -37,4 +37,5 @@
     path('articles/<slug>/comment/<int:id>/',
          comments_list, name='comment_an_article'),
     path('articles/<slug>/comment/', comments_details, name='modify_a_comment'),
+    path('articles/<slug>/report', ReportCreateAPIView.as_view()),
```

Comment:
Do we have a standard for urls? All other urls have a trailing slash

Reply:
corrected